.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+on:
+  push:
+    branches:
+      - '**'
+      - '!master'
+jobs:
+  build:
```

Comment:
Could you add a `name` here, something like: `Build & Test`.

Reply:
Done. https://github.com/hCaptcha/vue-hcaptcha/actions/runs/438466102